src/NumberTheory/GaloisGrp/GaloisGrp.jl

```diff
@@ -2664,7 +2664,11 @@ function blow_up(G::PermGroup, C::GaloisCtx, lf::Vector, con::PermGroupElem=one(
 
   icon = inv(con)
 
-  gs = map(Vector, gens(G))
+  if ngens(G) == 0
+    gs = Vector{Int}[]
+  else
+    gs = map(Vector, gens(G))
+  end
```

Comment:
```suggestion
  gs = Vector.(gens(G))
```
this infers as `Vector{Vector{Int64}}` as well and is a bit easier to read

Reply:
That's interesting, but I still would like to understand why `map(Vector, gens(G))` doesn't work anymore.

Reply:
Best would be to explicitly specify the return type, then Julia does not have to guess. That is, either
```suggestion
  gs = map(Vector{Int}, gens(G))
```
or
```suggestion
  gs = Vector{Int}.(gens(G))
```